src/lib/airtable/request.js

```diff
@@ -225,8 +226,26 @@ export const createManyInventorys = async (records) => {
   return Promise.all(createPromises);
 };
 
-export const createPurchaseRequest = async (record) => {
-  return createRecord(Tables.PurchaseRequests, record);
+// NONGENERATED: Create an inventory record (add product to site)
+export const createPurchaseRequest = async (purchaseRequest) => {
```

Comment:
I'm not really confident this warrants a custom endpoint. Thoughts?

Reply:
I don't think this warrants a custom endpoint. In the event that we want to do admin authentication (checking the `reviewerId` is an admin, which we should do) on the backend, We can write it in the `write` access resolver for purchase requests.

Details on how to do that with airlock / access resolvers is on: [this](https://share.descript.com/view/mu2V9m33Qa4) demo at around 8:46. If you wanna skip ahead via the transcript, you can CMD+F "So if your rules differ depending on whether or not your users are reading or writing to a base"